src/App.tsx

```diff
@@ -1,10 +1,16 @@
-
-import './App.css';
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import "./App.css";
+import { Homepage } from "./components/Homepage/Homepage";
+import { ROUTES } from "./constants/routes";
 
 function App() {
   return (
     <div className="App">
- 
+      <Router>
+        <Routes>
+          <Route path={ROUTES.HOME} element={<Homepage />} />
+        </Routes>
+      </Router>
```

Comment:
This is the typical (but far from only) way we declare routes in React. Right now, we only have a homepage. In the next PR, we'll create several more routes that look like the one on line 11.